### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by docwilmot**
As requested.

### CODE REVIEW

File: sites/all/themes/backdropapi/style.css [hunk 1]
```diff
@@ -5,3 +5,18 @@ td.active {
...
+h2 {
+  font-weight: 500;
+}
+code {
+  border: none;
+  background-color: #F7F7F7;
+  margin-bottom: 2em;
+}
+
+blockquote.d7diff {
```
Comment (quicksketch) [inline sites/all/themes/backdropapi/style.css]: Hi @docwilmot, this looks good to me. Could we make this a bit more generic a class so we could use it in the future for Backdrop 1.x to 2.x changes? How about just "change" or "diff"?

Comment (docwilmot) [inline sites/all/themes/backdropapi/style.css]: I think its best to keep it so we can style D7 differences differently from other differnces, if you understand what I mean. In time we could create a .bddiff callout block, give it a red background so its obvious etc.

**Comment by docwilmot** [comment]
Reminder.

**Comment by quicksketch** [comment]
Hm, I'm still not hot on the class name. We can revisit this later I suppose. I'll merge and pull this into the live API site.

**Pull request merged**